libusb/os/windows_winusb.c

```diff
@@ -1517,11 +1504,11 @@ static int winusb_get_device_list(struct libusb_context *ctx, struct discovered_
 	unsigned int unref_size = UNREF_SIZE_STEP;
 	unsigned int unref_cur = 0;
 
-	// PASS 1 : (re)enumerate HCDs (allows for HCD hotplug)
-	// PASS 2 : (re)enumerate HUBS
-	// PASS 3 : (re)enumerate generic USB devices (including driverless)
+	// PASS 1 : (re)enumerate HUBS
+	// PASS 2 : (re)enumerate master USB devices that have a device interface
+	// PASS 3 : (re)enumerate HCDs (allows for HCD hotplug)
+	// PASS 4 : (re)enumerate generic USB devices (including driverless)
 	//           and list additional USB device interface GUIDs to explore
-	// PASS 4 : (re)enumerate master USB devices that have a device interface
 	// PASS 5+: (re)enumerate device interfaced GUIDs (including HID) and
```

Comment:
Isn't HID pass 5 and EXT pass 6+ ?

Reply:
Actually since you are not changing the number of passes (in a previous version you had removed the HCD pass if I remember correctly) you are not obliged to fix up these comments :) We can do that later, to simplify this PR.

Reply:
Indeed, changed with last commit